www/contact.php

```diff
@@ -7,63 +7,58 @@
 
 <?php HTML_HEADER(); ?>
 
-<main role="main">
-<section class="section contact-info-section" id="contacts">
-  <div class="contact-info">
-    <h2>Контакты</h2>
-    <p class="contact-info-email">
-      <a href="mailto:info@vibrobox.com">info@vibrobox.com</a>
-    </p>
-    <div class="info-block">
-      <h3>Главный офиc:</h3>
-      <p>
-        <span>VibroBox OÜ</span>
-        <span>Бульвар Равала 8, офис 810, Таллин, 10143, Эстония</span>
-      </p>
-      <p class="phone-group">
-        <span>
-        Тел.:<a href="tel:+372 602 7122">+372 602 7122</a>
-        </span>
-      </p>
-    </div>
-    <div class="info-block">
-      <h3>Центр разработки:</h3>
-      <p>
-        <span>ООО «Сител»</span>
-        <span>Логойский тракт 22A/2, офис 402, г. Минск, 220090, Беларусь</span>
-      </p>
-      <p class="phone-group">
-        <span>
-        Тел.:<a href="tel:+375 29 176 83 35">+375 29 176 83 35</a>
-        </span>
-      </p>
-    </div>
-    <footer class="footer">
-      <p class="copyright">VibroBox © 2015-2017</p>
-    </footer>
-  </div>
-  <!-- Map is rendered here by leaflet and our map.js script. -->
-  <div class="map-canvas" id="map-canvas"></div>
+<h1 class="contacts-h1">Наши контакты</h1>
+
+<p class="contacts-intro">
+  <?= T('contactsIntro') ?>
+</p>
+
+<main role="main" class="contacts">
+
+<section class="contacts__offices">
+  <article class="contacts__office">
```

Comment:
Мне кажется, что article не совсем правильно используется в данном контексте. section or div tag сюда лучше подошли бы.

> The article element represents a complete, or self-contained, composition in a document, page, application, or site and that is, in principle, independently distributable or reusable, e.g. in syndication. This could be a forum post, a magazine or newspaper article, a blog entry, a user-submitted comment, an interactive widget or gadget, or any other independent item of content.  ( https://www.w3.org/TR/html5/sections.html#the-article-element )